autoload/lsp/omni.vim

```diff
@@ -154,8 +154,9 @@ function! lsp#omni#get_vim_completion_item(item) abort
         let l:word = a:item['label']
         let l:abbr = a:item['label']
     endif
+    let l:info = has_key(a:item, 'detail') ? a:item['detail'] : ''
     let l:menu = lsp#omni#get_kind_text(a:item)
-    let l:completion = { 'word': l:word, 'abbr': l:abbr, 'menu': l:menu, 'icase': 1, 'dup': 1 }
+    let l:completion = { 'word': l:word, 'abbr': l:abbr, 'menu': l:menu, 'info': l:info, 'icase': 1, 'dup': 1 }
```

Comment:
Can we instead change this to the following to avoid unnecessary string allocations?

```vim
if has_key(a:item, 'detail')
  let l:completion['detail'] = a:item['detail']
endif
```

Reply:
Yes, that makes more sense to me.